src/lib/instanceManager/createInstanceManager.js

```diff
@@ -35,9 +35,9 @@ module.exports = ({
       instanceByName[name] = instance;
 
       const computedEdgeConfigId =
-        (turbine.buildInfo.environment === "development" &&
+        (turbine.environment.stage === "development" &&
```

Comment:
I think it's weird that they named it `stage` :)

Reply:
You and me both. I was thinking stage as in staging, but it is stage as in development stage.